linera-service/src/cli_wrappers/kind.rs

```diff
@@ -11,12 +11,9 @@ pub struct KindCluster {
 }
 
 impl KindCluster {
-    pub async fn new(cluster_id: Option<u32>) -> Result<Self> {
-        let cluster = match cluster_id {
-            Some(id) => Self { id },
-            None => Self {
-                id: Self::get_random_cluster_id(),
-            },
+    pub async fn new() -> Result<Self> {
```

Comment:
`KindCluster` should also have a context (see commend for `KubectlWrapper`).

Reply:
Commented on this below